source/_components/twentemilieu.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: "Twente Milieu"
+description: "Instructions on how to integrate Twente Milieu with Home Assistant."
+logo: twentemilieu.png
+ha_category:
+  - Sensor
+  - Environment
+ha_release: 0.97
+ha_iot_class: Cloud Polling
+ha_config_flow: true
```

Comment:
I had forgotten we had this key. Do we use it somewhere?

Reply:
It shows up in the sidebar on the integration page.
Currently working getting the set of properties into sync with the docs website, goal: Able to sync up with the code base.

After that, we can start adding more visual elements to the website (and hopefully filters and other stuff).